otbenchmark/ReliabilityProblem110.py

```diff
@@ -31,19 +31,19 @@ def __init__(self, threshold=0.0, mu1=0.0, sigma1=1.0, mu2=0.0, sigma2=1.0):
             else :
                 g2 = 0.5 - 0.1 * x2
 
-        We have x1 ~ Normal(mu1, sigma1) and x2 ~ Normal(mu2, sigma2).
+        We have x1 ~ Normal(mu[0], sigma[0]) and x2 ~ Normal(mu[1], sigma[1]).
 
         Parameters
         ----------
         threshold : float
             The threshold.
-        mu1 : float
+        mu[0] : float
```

Comment:
This description should be shortened into:

```
mu : sequence of floats
    The list of two items representing the means of the gaussian distributions.
```

Please apply this change to the other classes as well.

Reply:
for sigma, I find a problem with flake8 when I write "The list of two items representing the standard deviations of the gaussian distributions", because the line is too long.  it is possible to write just :
"The list of the standard deviations of the gaussian distributions"

Reply:
Please keep the previous long comment and split it into two lines: flake8 does not forbid this.